inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/scatter_update.cpp

```diff
@@ -0,0 +1,149 @@
+        ::testing::ValuesIn(idxShapes),
+        ::testing::ValuesIn(idxValue),
+        ::testing::ValuesIn(updateShapes_1),
+        ::testing::ValuesIn(axisValue_1),
+        ::testing::ValuesIn(inputPrecisions),
+        ::testing::ValuesIn(idxPrecisions),
+        ::testing::Values(CommonTestUtils::DEVICE_CPU)
+);
+
+//////////////////////5D////////////////////////////////////////////
+const std::vector<std::vector<size_t>> inputShapes5D = {
+        {20, 12, 22, 23, 17}
+};
+const std::vector<std::vector<size_t>> idxShapes1D = {
+        {6}
+};
+const std::vector<std::vector<size_t>> idxValue1D = {
+        {2, 3, 4, 9, 0, 10}
+};
+
+const std::vector<int> axisValue5D_2 = {2};
+const std::vector<std::vector<size_t>> updateShapes5D_2 = {
+        {20, 12, 6, 23, 17}
+};
+
+const std::vector<int> axisValue5D_3 = {3};
+const std::vector<std::vector<size_t>> updateShapes5D_3 = {
+        {20, 12, 22, 6, 17}
+};
+
+const std::vector<int> axisValue5D_4 = {4};
+const std::vector<std::vector<size_t>> updateShapes5D_4 = {
+        {20, 12, 22, 23, 6}
+};
+
+const std::vector<int> axisValue5D_N = {-2};
+const std::vector<std::vector<size_t>> updateShapes5D_N = {
+        {20, 12, 22, 6, 17}
+};
+
+const auto ScatterUpdate_5D_AXIS2 = ::testing::Combine(
+        ::testing::ValuesIn(inputShapes5D),
+        ::testing::ValuesIn(idxShapes1D),
+        ::testing::ValuesIn(idxValue1D),
+        ::testing::ValuesIn(updateShapes5D_2),
+        ::testing::ValuesIn(axisValue5D_2),
+        ::testing::ValuesIn(inputPrecisions),
+        ::testing::ValuesIn(idxPrecisions),
+        ::testing::Values(CommonTestUtils::DEVICE_CPU)
+);
+const auto ScatterUpdate_5D_AXIS3 = ::testing::Combine(
+        ::testing::ValuesIn(inputShapes5D),
+        ::testing::ValuesIn(idxShapes1D),
+        ::testing::ValuesIn(idxValue1D),
+        ::testing::ValuesIn(updateShapes5D_3),
+        ::testing::ValuesIn(axisValue5D_3),
+        ::testing::ValuesIn(inputPrecisions),
+        ::testing::ValuesIn(idxPrecisions),
+        ::testing::Values(CommonTestUtils::DEVICE_CPU)
+);
+const auto ScatterUpdate_5D_AXIS4 = ::testing::Combine(
+        ::testing::ValuesIn(inputShapes5D),
+        ::testing::ValuesIn(idxShapes1D),
+        ::testing::ValuesIn(idxValue1D),
+        ::testing::ValuesIn(updateShapes5D_4),
+        ::testing::ValuesIn(axisValue5D_4),
+        ::testing::ValuesIn(inputPrecisions),
+        ::testing::ValuesIn(idxPrecisions),
+        ::testing::Values(CommonTestUtils::DEVICE_CPU)
+);
+const auto ScatterUpdate_5D_NEG = ::testing::Combine(
+        ::testing::ValuesIn(inputShapes5D),
+        ::testing::ValuesIn(idxShapes1D),
+        ::testing::ValuesIn(idxValue1D),
+        ::testing::ValuesIn(updateShapes5D_N),
+        ::testing::ValuesIn(axisValue5D_N),
+        ::testing::ValuesIn(inputPrecisions),
+        ::testing::ValuesIn(idxPrecisions),
+        ::testing::Values(CommonTestUtils::DEVICE_CPU)
+);
+// open after ngraph reference implementation merged
```

Comment:
Nobody is working on them now. But the work on finding the person to implement them is in progress.

Reply:
If nobody implements reference in NGraph, we should do it ourselves. I am talking about the JIRA task description:
```
* Add test definition to the shared part of plugin functional tests
* Create instance for CPU plugin
* Create Reference in Ngraph
```

@lazarevevgeny Is it possible to find a person who will implement the reference in Ngraph? If not, @chenhu-wang @dmitry-gorokhov Could you implement the reference?

Reply:
@iefode Which one JIRA task? 
The PR covers only following tickets: 29131, 30364, 26814 which assumes only CPU implementation. I am against blocking this PR because of lack of NG reference. 
We will discuss who will be able to implement reference but it will be done separately.

Reply:
33266 -> `[IE TESTS] Add to the singe layer tests "ScatterUpdate"`. I am OK to do it separately

Reply:
I agree with @dmitry-gorokhov, let's add nGraph reference implementation in a separate PR. But please wait when @rkazants check the model e2e.

@rkazants , when do you plan to finish the check?

Reply:
> I agree with @dmitry-gorokhov, let's add nGraph reference implementation in a separate PR. But please wait when @rkazants check the model e2e.
> 
> @rkazants , when do you plan to finish the check?

today - earlier tomorrow

Reply:
@rkazants , just a kind reminder, better to use latest commit to avoid regression check.

Reply:
> @rkazants , just a kind reminder, better to use latest commit to avoid regression check.

Good news, e2e test for Wide and Deep model with ScatterNDUpdate passed. Thanks a lot!